customer/tests/factories.py

```diff
@@ -17,5 +17,10 @@ class Meta:
 
     # Others
     additional_information = Faker("paragraph", nb_sentences=5)
+    has_children = Faker("boolean")
+    has_hitas_ownership = Faker("boolean")
+    is_age_over_55 = Faker("boolean")
+    is_right_of_occupancy_housing_changer = Faker("boolean")
     last_contact_date = Faker("date")
     primary_profile = SubFactory(ProfileFactory)
+    right_of_residence = fuzzy.FuzzyInteger(1, 1000000000)
```

Comment:
I think this `fuzzy` stuff is [deprecated](https://factoryboy.readthedocs.io/en/stable/fuzzy.html).

Reply:
Changed to Faker